fhs/pkg.yaml

```diff
@@ -1,14 +1,26 @@
 name: fhs
+variant: scratch
+shell: /bin/bash
+dependencies:
+  - stage: base
 steps:
 - install:
   - |
     PREFIX=/rootfs
-    mkdir -pv ${PREFIX}/{dev,lib,proc,sys,bin,sbin,etc,opt,run}
+    mkdir -pv ${PREFIX}/{dev,proc,sys,etc,opt,run}
+    mkdir -pv ${PREFIX}/run/lock
     mkdir -pv ${PREFIX}/usr/{bin,sbin,lib,var,include,src}
-    mkdir -pv ${PREFIX}/usr/local/{bin,sbin,lib,include}
-    mkdir -pv ${PREFIX}/var/{log,run,mail,spool}
+    mkdir -pv ${PREFIX}/var/{log,mail,spool}
     install -dv -m 0750 ${PREFIX}/root
     install -dv -m 1777 ${PREFIX}/tmp ${PREFIX}/var/tmp
+    cd ${PREFIX}
+    ln -sT usr/lib lib64
+    ln -sT usr/lib lib
+    ln -sT usr/bin bin
+    ln -sT usr/sbin sbin
```

Comment:
Merging bin and sbin, as e.g. Arch Linux did, is possible, but is a bit more difficult (as more packages install there) and not necessary to do in this PR since it can build without that